avocado/utils/memory.py

```diff
@@ -320,3 +320,16 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+def memtotal_sys():
+    """
+    get the total memory as detected by system
```

Comment:
Hello @sathnaga, always avid to find out which new area are you going to teach me about with each new PR. Would you please give some details on what is this actually good for? I found:

                The /sys/devices/system/memory contains a snapshot of the
		internal state of the kernel memory blocks. Files could be
		added or removed dynamically to represent hot-add/remove
		operations.

On my machine I get `16515072` kb, which is a bit more than I get from `/proc/meminfo` 16075160. I'm wondering why the difference and what is this information useful to compare to the MemTotal entry in meminfo?

As for the style, I wanted to ask you to use python, but the `grep` approach does not looks that bad as trying to do the same/similar in python so please just replace the `cat` with `open(..., "r").read()` and please expand the docstring as people might accidentally use this function even thought they might be looking for another information.